.github/workflows/run_test_suite.yaml

```diff
@@ -82,7 +82,10 @@ jobs:
         with:
           command: test
           args: --features test_kubo,headers
-
+      - name: 'Run C tests'
```

Comment:
Can we make this it's own "job" (as opposed to a "step" within a "job")? This way it will show up as its own line in the PR status check area.

Reply:
yeah!

Reply:
Haven't yet looked into caching yet, but a separate job results in a whole new env install and recompile; maybe rebranded as "FFI integration tests" would be more appropriate as a step within `run-test-suite-linux`?

Reply:
I think it's good for us to split all the distinctive test runs out. Eventually I would like us to be able to run them as stand-alone workflows.

Reply:
alternatively, these could also live as `rust/noosphere/tests/*.c` bootstrapped by cargo test, testing the rust library moreso than a bindings environment